assets/javascripts/discourse/custom-wizard-admin-route-map.js.es6

```diff
@@ -11,6 +11,10 @@ export default {
       this.route('adminWizardsApis', { path: '/apis', resetNamespace: true }, function() {
         this.route('adminWizardsApi', { path: '/:name', resetNamespace: true });
       });
+
+      this.route('adminWizardsTransfer',{path:'/transfer', resetNamespace:true});
```

Comment:
Nitpicking here, but please follow the same formatting as the existing code :)
```
this.route('adminWizardsTransfer', { path: '/transfer', resetNamespace: true });
```
(and remove the blank lines)